pantheon-bundle/frontend/src/app/versions.tsx

```diff
@@ -744,6 +744,13 @@ class Versions extends Component<IProps, IState> {
                 })
         }
     }
+
```

Comment:
I have one query here, as we already know, content type can be either module or assembly, we can compare if content type is one of them other wise return. what do you suggest ?